css/style.scss

```diff
@@ -0,0 +1,460 @@
+@import '../../../themes/magentacloud25/core/css/functions';
+@import '../../../themes/magentacloud25/core/css/theme';
+@import '../../../themes/magentacloud25/core/css/icons';
+
+// $bg_color_unset: unset;
+$red_color: #FF0000;
+
+#hello {
+	color: $red_color;
+}
+
+/* slide - forms  */
+/*pagination*/
+#nmc_welcome_popup {
+	.modal-wrapper {
+		.nmc_welcome_popup-header {
+			h2 {
+				float: left;
+				color: var(--scl-color-grey-90)!important;
+				font-weight: bold!important;
+				padding: 0!important;
```

Comment:
We can remove " !important " from everywhere here as this is our custom application and we can modify existing component styling instead of overwriting

Reply:
removed all !important in styles